code/modules/mob/living/carbon/carbon_update_icons.dm

```diff
@@ -1,7 +1,3 @@
-/mob/living/carbon/update_obscured_slots(obscured_flags)
-	..()
-	update_body()
```

Comment:
Already called by the appropriate clothing procs whenever needed :/

Reply:
In testing removing this line will break chameleon gear

Reply:
oh wow, good catch

Reply:
I don't have the energy to debug this, and I won't for quite a while. 
I have taken the cowards way out and reverted this specific change